### ISSUE TITLE
Unable to add testID or accessibilityLabel in the Dialog.

### ISSUE THREAD

**Issue by tkyr-hh** [douglasjunior/react-native-simple-dialogs#70]
### - React Native Info
System:
    OS: macOS 14.2.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 239.30 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.19.0 - /usr/local/bin/node
    Yarn: 1.22.21 - /usr/local/bin/yarn
    npm: 10.2.3 - /usr/local/bin/npm
    Watchman: 2023.12.04.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.14.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 23.2, iOS 17.2, macOS 14.2, tvOS 17.2, visionOS 1.0, watchOS 10.2
    Android SDK:
      API Levels: 29, 30, 31, 33
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 30.0.3, 31.0.0, 33.0.0, 33.0.1
      System Images: android-23 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom_64, android-31 | Google APIs Intel x86 Atom_64, android-31 | Google Play Intel x86 Atom_64, android-33 | Google APIs Intel x86 Atom_64
      Android NDK: Not Found
  IDEs:
    Android Studio: 2023.1 AI-231.9392.1.2311.11076708
    Xcode: 15.2/15C500b - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.21 - /usr/local/opt/openjdk@11/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0
    react-native: 0.71.14 => 0.71.14
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### - What code snippet did you use?

```
import Dialog from "react-native-simple-dialog";
...
<Dialog
      overlayStyle={styles.overlay}
      visible={isVisible}
      dialogStyle={ styles.dialog}
      animationType="slide"
      onTouchOutside={_onTouchOutside}
      testID={testIDs.LOGIN_MODAL}>
....
</Dialog>

```

**Comment by douglasjunior** [comment]
Released in version 2.0.2, thank you.
